zowe/zowe-cli
# feat: Allow 'false' as a proper value for credentialManagerOverride APIs

**What It Does**

Allows credential manager override to specify a value of `false` 

**How to Test**

- Download this VSIX: https://ent.box.com/s/5t9a90ewumbalqoj75too7vgyeehix7l
- Disable the `zowe.security.secureCredentialsEnabled` property
- Remove the secure properties from the `secure[]` array.
- Attempt to store credentials (`right-click > Manage Profile > Update Credentials`)
- You should see a warning about credentials being stored in plain-text
  <img width="252" alt="image" src="https://github.com/user-attachments/assets/a51ad8a4-9ad6-467b-8d1a-41bb22ea5d01">



**Review Checklist**
I certify that I have:


**Additional Comments**

This is related to the following Zowe Explorer issue:
- https://github.com/zowe/zowe-explorer-vscode/issues/2622
